2021-03-09 13:45:54,193 INFO    MainThread:13924 [wandb_setup.py:_flush():69] setting env: {}
2021-03-09 13:45:54,193 INFO    MainThread:13924 [wandb_setup.py:_flush():69] setting login settings: {}
2021-03-09 13:45:54,193 INFO    MainThread:13924 [wandb_init.py:_log_setup():334] Logging user logs to C:\Users\Arnesh Kumar Bose\Desktop\CS6910 Assignments\wandb\run-20210309_134554-oobuy1jl\logs\debug.log
2021-03-09 13:45:54,193 INFO    MainThread:13924 [wandb_init.py:_log_setup():335] Logging internal logs to C:\Users\Arnesh Kumar Bose\Desktop\CS6910 Assignments\wandb\run-20210309_134554-oobuy1jl\logs\debug-internal.log
2021-03-09 13:45:54,193 INFO    MainThread:13924 [wandb_init.py:init():367] calling init triggers
2021-03-09 13:45:54,197 INFO    MainThread:13924 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'n_layers': 3, 'hidden_layer_size': 64, 'learn_rate': 0.001, 'batch_size': 16, 'epochs': 10, 'alpha': 0.0005, 'optimizer': 'gradient_descent', 'activation': 'sigmoid', 'weight_init': 'random'}
2021-03-09 13:45:54,197 INFO    MainThread:13924 [wandb_init.py:init():389] re-initializing run, found existing run on stack: vtb1m9r5
2021-03-09 13:45:54,197 INFO    MainThread:13924 [wandb_run.py:finish():1135] finishing run arneshbose1/deep_learning_assign2/vtb1m9r5
2021-03-09 13:45:54,197 INFO    MainThread:13924 [jupyter.py:save_history():320] not saving jupyter history
2021-03-09 13:45:54,197 INFO    MainThread:13924 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-03-09 13:45:54,197 INFO    MainThread:13924 [wandb_init.py:_jupyter_teardown():266] cleaning up jupyter logic
2021-03-09 13:45:54,197 INFO    MainThread:13924 [wandb_run.py:_atexit_cleanup():1446] got exitcode: 0
2021-03-09 13:45:54,201 INFO    MainThread:13924 [wandb_run.py:_restore():1418] restore
